### ISSUE TITLE
neofetch hangs trying to load "shell" line when remoted into Windows 11 over openSSH

### ISSUE THREAD

**Issue by deltamualpha** [dylanaraps/neofetch#2053]
## Description

When running in Powershell 7 when connected over OpenSSH on Windows 11, `neofetch` hangs during what I assume is an attempt to determine what shell it's running under. Same behavior seen in default `cmd` shell as well.

`neofetch` is installed in this case using `scoop`.

Also, when run locally on the machine, the shell is misidentified as Bash, instead of Powershell 7.

#### Neofetch version

7.1.0

## Screenshot

Remote:
<img width="751" alt="Screen Shot 2022-02-13 at 10 42 29 PM" src="https://user-images.githubusercontent.com/2105458/153796517-c79f1d9b-ff49-45eb-825b-5fb2d7c11433.png">

Local:
<img width="809" alt="Screen Shot 2022-02-13 at 10 44 12 PM" src="https://user-images.githubusercontent.com/2105458/153796653-3ad8bef6-11da-43c5-ac73-bce742351b0c.png">

## Verbose log

https://gist.github.com/deltamualpha/3b2e03cda1ab358eb1bcefeeb0976dd2

**Comment by deltamualpha** [comment]
Aha, found the logged output. Attached what appears to be the relevant part here: https://gist.github.com/deltamualpha/3b2e03cda1ab358eb1bcefeeb0976dd2

**Comment by whentojump** [comment]
I had similar issues. I found that the core problem is: _when SSH'ing into a Windows machine, **Git Bash** will somehow get a different `$SHELL` than the local case._

Here is my output:

Local:

```shell
$ echo $SHELL
/usr/bin/bash
```

Remote:

```shell
$ echo $SHELL
/c/windows/system32/windowspowershell/v1.0/powershell.exe
```

(Note: mine is PowerShell rather than CMD like in OP, because I did set it so, as the Windows "login shell", guided by [this document](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_server_configuration#configuring-the-default-shell-for-openssh-in-windows))

Since neither PowerShell or CMD supports `--version`, things will go unexpectedly.

I don't know what a long-term solution could be, **but a quick fix is**: in `config.conf`, change this line to:

```
shell_version="off"
```

**Comment by deltamualpha** [comment]
Good catch! For me, it would still hang until I just commented out the Shell line entirely. I wonder if it's trying to launch a subshell by accident?
